requests/src/main/scala/weco/api/requests/responses/LookupPendingRequests.scala

```diff
@@ -21,16 +25,16 @@ trait LookupPendingRequests extends CustomDirectives {
         userHolds <- sierraService.getStacksUserHolds(patronNumber)
 
         holdsWithCatalogueIds <- Future.sequence(
-          userHolds.right.get.holds.map { hold =>
+          userHolds.right.get.holds.map { hold: StacksHold =>
             itemLookup
               .bySourceIdentifier(hold.sourceIdentifier)
               .map {
-                case Left(elasticError) =>
-                  warn(s"Unable to look up $hold in Elasticsearch")
+                case Left(elasticError: ElasticsearchError) =>
```

Comment:
Previously we just chucked this error, not sure why - i've added it to the warning here.